### ISSUE TITLE
Lines between points

### ISSUE THREAD

**Issue by Michele64** [redfish64/TinyTravelTracker#51]
Thanks for this gem! I'm using it without interruption since version 1.0.12 and always happywith it,but
there are no lines between points since the transfer to mapzen, using the newest version from f-droid.

**Comment by fdroiduser** [comment]
I'm most happy without the lines between points :-)
So if the developer decides to put them back, please add an option to enable/disable these lines!

Thank you

**Comment by davidmef** [comment]
I have been using the application since June 2016 and I have never seen the lines between points except in the documentation. I would like to have them since it is complicated to understand my route just from differences in color.

**Comment by Michele64** [comment]
Exactly my point. The lines disappeared since moving the maps source to mapzen (I think that's unrelated, just defining a point in time)

**Comment by fdroiduser** [comment]
Yes, you are right, they disappeared with the same release as the change of tiles.
However, I repeat that on my side I'm happy _without_ the lines, so I suppose the best would be an option do enable/disable them. (That's probably just little work?)
Which side should be default (with or without lines) is completely equal to me.
